packages/web-api/src/WebClient.spec.js

```diff
@@ -34,20 +34,6 @@ describe('WebClient', function () {
       assert.instanceOf(client, WebClient);
       assert.notExists(client.axios.defaults.headers.Authorization);
     });
-    it('should not modify global defaults in axios', function () {
```

Comment:
I didn't feel like this test offers much value and since this PR moves from axios 0.x to 1.x, also not sure how much we should be testing the output of _other_ packages.

Reply:
@filmaj As you can see, the context behind this unit test is https://github.com/slackapi/node-slack-sdk/issues/1037 This package had an issue polluting the global singleton instance of `axios` package. This test tries to prevent the same issue from occurring again. If axois 1.x does not allow such destructive operations to the global singleton, we can safely remove this test. Thus, I don't think we're testing 3rd party library here. We verify our package works without any major side effects.

If investigation on how to properly maintain this test with axios 1.x could require our time and efforts, it's fine to delete the test anyways. Whenever we have something similar in the future, we can add tests again.

Reply:
It seems [in the latest axios, no specific value is set for the HTTP POST default headers](https://github.com/axios/axios/blob/a48a63ad823fc20e5a6a705f05f09842ca49f48c/lib/defaults/index.js#L155-L157). This is [also the case for 'common' HTTP all-verb headers](https://github.com/axios/axios/blob/a48a63ad823fc20e5a6a705f05f09842ca49f48c/lib/defaults/index.js#L150).

[In v0.27.2 of axios, this was not the case: POST would set a default `Content-Type`](https://github.com/axios/axios/blob/v0.27.2/lib/defaults/index.js#L142-L144). and [the default was `application/x-www-form-urlencoded`](https://github.com/axios/axios/blob/v0.27.2/lib/defaults/index.js#L9-L11).

Therefore, I think this test can be safely removed.

Reply:
Thanks for checking this!